f21/ps.html

```diff
@@ -1025,33 +1021,164 @@ <h2><u> Lecture / Lab 7</u></h2>
 
 
 
-   <li > <b>Due Date: 8 November</b>
+   <li > <b>Due Date: 3 November</b>
 <b  style="float:right">Reading: Think CS:  <a href="https://runestone.academy/runestone/books/published/thinkcspy/Selection/ConditionalExecutionBinarySelection.html" target="_blank">Chapter 7.4</a>, <a href="https://docs.python.org/3.3/library/turtle.html?highlight=turtle" target="_blank"> Turtle Graphics</a> & <a href="lab6.html" target="_blank">Lab 6</a> </b><br>
 
 <p>
 <b><p><b>Review from Lecture/Lab 4</b></b>
 
-<p>The program, <tt>turtleString.py</tt> (available at:  <a href="https://github.com/HunterCSci127/CSci127"target="_blank">
-	https://github.com/HunterCSci127/CSci127</a>) takes a string as input and uses that string to control what the turtle draws on the screen (inspired by <a href="https://code.org/curriculum/course2/1/Teacher">code.org's graph paper programming</a>).  Currently, the program processes the following commands:
+<p>The program, <tt>mazeRunner.py</tt> (available at:  <a href="https://github.com/HunterCSci127/CSci127"target="_blank">
+	https://github.com/HunterCSci127/CSci127</a>) takes a string as input and uses that string to control where the user goes throughout the maze (inspired by <a href="https://code.org/curriculum/course2/1/Teacher">code.org's graph paper programming</a>).
+        <p>
+        Modify the function playGame() to allow the user use the following symbols:
         <ul>
-          <li> 'F':  moves the turtle forward 50 steps
-          <li> 'L':  turns the turtle 90 degrees to the left
-          <li> 'R':  turns the turtle 90 degrees to the right
-          <li> '^':  lifts the pen
-          <li> 'v':  lowers the pen
-		      <li> 'r':	 change turtle color to red
-          <li> 'g':  change turtle color to green
-		      <li> 'b':  change turtle color to blue
+          <li> 'R':  go Right of the current location
+          <li> 'L':  go Left of the current location
+          <li> 'U':  go Up from the current location
+	  <li> 'D':  go Down from the current location
         </ul>
-        For example, if the user enters the string "FLFLFLFL^FFFvFLFLFLFL", the turtle would move forward and then turn left.  It repeats this 4 times, drawing a square.  Next, it lifts the pen and move forward 3, puts the pen back down and draw another square.
+	
+	<p>
+	You must check first if each command is a valid move, utilizing the given functions, then update the location of the User (thru the given variables 'row' and 'col'). If the user attempts to go through walls or simply does not reach the end of the maze after all of their commands have been executed, return false. If they reach the end (represented by an <tt>'X'</tt> at the location), return True.
```

Comment:
Let's use through instead of thru consistently. Thanks!

Reply:
Oh yes, that slipped by me. Got it.